domains/certificates/Query.API/API/Query.API/ApiModels/Requests/CreateContract.cs

```diff
@@ -13,10 +14,19 @@ public class CreateContract
     /// <summary>
     /// Starting date for generation of certificates in Unix time seconds
     /// </summary>
+    [JsonPropertyName("startDate")]
     public long StartDate { get; init; }
 
     /// <summary>
     /// End date for generation of certificates in Unix time seconds. Set to null for no end date
     /// </summary>
+    [JsonPropertyName("endDate")]
     public long? EndDate { get; set; }
+
+    /// <summary>
+    /// Metering point type. Can be either Production or Consumption
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    [JsonPropertyName("meteringPointType")]
+    public MeteringPointType? MeteringPointType { get; set; }
```

Comment:
Why must the caller send the metering point type?

Reply:
removed